recipes/triqs_ctseg/conda_build_config.yaml

```diff
@@ -0,0 +1,11 @@
+mpi:
+  - mpich
+  - openmpi
+mpich:
+  - '4'
+openmpi:
+  - '5'
```

Comment:
These lines might be the important difference between this and your other feedstock https://github.com/conda-forge/triqs_cthyb-feedstock/blob/7a4b637fa3928155cdeaebdb2c2a4f2a934376b0/recipe/conda_build_config.yaml